docs/domainmetadata.rst

```diff
@@ -18,6 +18,17 @@ documentation.
 Apart from raw SQL statements, setting domain metadata can be done with
 ``pdnsutil set-meta`` and retrieving metadata is done with ``pdnsutil get-meta``.
 
+The following options can only be read via the HTTP API metadata endpoint:
```

Comment:
```suggestion
The following options can only be read (not written to) via the HTTP API metadata endpoint.
```

The original working was unclear: does "only be read via the HTTP API..." mean:
a) you can't read them using pdnsutil
b) you can't write them using the HTTP API